dolphinscheduler-standalone-server/src/main/resources/application.yaml

```diff
@@ -216,7 +216,7 @@ worker:
   server-load-protection:
     enabled: true
     # Worker max cpu usage, when the worker's cpu usage is smaller then this value, worker server can be dispatched tasks.
-    max-cpu-usage-percentage-thresholds: 0.9
+    max-cpu-usage-percentage-thresholds: 1.5
```

Comment:
```suggestion
    max-cpu-usage-percentage-thresholds: 1.0
```

Reply:
@ruanwenjun hello  Sometimes the utilization rate of underlying resources can be greater than 1, so setting it to 1.5 can overload and cause e2e failure if set to 1
![image](https://github.com/apache/dolphinscheduler/assets/127465317/376c0ec1-18e9-4c0f-8797-15cbdcfebc74)

Reply:
This should be a bug, we may need to use getTotalCpuUsedPercentage() to compare with maxCpuUsagePercentageThresholds at ServerLoadProtection